libs/domain/product/references/src/references.component.ts

```diff
@@ -0,0 +1,35 @@
+import { resolve } from '@spryker-oryx/di';
+import { LayoutMixin } from '@spryker-oryx/experience';
+import {
+  AlternativeProductsListService,
+  ProductMixin,
+} from '@spryker-oryx/product';
+import { computed, hydratable } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { of } from 'rxjs';
+
+@hydratable()
+export class ReferencesComponent extends ProductMixin(LayoutMixin(LitElement)) {
+  protected alternativeProductsListService = resolve(
+    AlternativeProductsListService
+  );
+
+  protected list = computed(() => {
```

Comment:
```suggestion
  protected $list = computed(() => {
```

we just agreed on a new convention to prefix both computed and signals with a dollar.